tests/python/driver/tvmc/test_compiler.py

```diff
@@ -29,6 +30,15 @@
 from tvm.driver import tvmc
 
 
+def vitis_ai_available():
+    """Return whether Vitis AI tools are available"""
+    pyxir_spec = importlib.util.find_spec("pyxir")
+    if not tvm.get_global_func("tvm.vitis_ai_runtime.from_xgraph", True) or pyxir_spec is None:
+        print("Skip because Vitis AI tools are not available")
```

Comment:
This `print` is probably not needed, as this is used within a `@pytest.mark.skipif`